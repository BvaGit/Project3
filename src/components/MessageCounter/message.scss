@import '../../utils/variables.scss';

.message-counter{
    margin-left: $margin_sm;
}