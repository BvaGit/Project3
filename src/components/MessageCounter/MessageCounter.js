import React from 'react';

import './message.scss';

const MessageCounter = () => (
    <div className="message-counter mention-pin">
        99..
    </div>
)

export default MessageCounter;