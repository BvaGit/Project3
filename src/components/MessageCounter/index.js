import MessageCounter from "./MessageCounter";

export default MessageCounter;