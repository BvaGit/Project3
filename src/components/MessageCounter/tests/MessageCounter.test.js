import React from 'react';
import { shallow, mount } from 'enzyme';
import MessageCounter from '../index';


describe('MessageCounter', () => {
    it("Should match snapshot", () => {
        const component = shallow(<MessageCounter />)
        expect(component.html()).toMatchSnapshot()
    })
})