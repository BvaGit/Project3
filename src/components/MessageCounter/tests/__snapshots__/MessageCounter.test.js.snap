// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageCounter Should match snapshot 1`] = `"<div class=\\"message-counter mention-pin\\">99..</div>"`;
